l1-contracts/contracts/common/libraries/Messaging.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.24;
+
+bytes32 constant BATCH_LEAF_PADDING = keccak256("zkSync:BatchLeaf");
+bytes32 constant CHAIN_ID_LEAF_PADDING = keccak256("zkSync:ChainIdLeaf");
+
+library Messaging {
```

Comment:
these methods are already present in `l1-contracts/contracts/common/libraries/MessageHashing.sol`